### ISSUE TITLE
How to reflect `extend-exclude` in pyproject.toml

### ISSUE THREAD

**Issue by agarichan** [charliermarsh/ruff-vscode#147]
Is it possible to have the editor reflect the `extend-exclude` setting in pyproject.toml?

Even where `extend-extend` is listed, there seems to be a warning on the editor.

```pyproject.toml
[tool.ruff]
select = ["E", "W", "F", "I", "B", "PL"]
extend-exclude = ["alembic"]
```

**Comment by VDuchauffour** [comment]
Same for me. Using `extend-exclude = ["tests/*.py"]` in a pyproject.toml implies no errors found with the Ruff CLI but still warnings on the editor.
Thanks

**Comment by charliermarsh** [comment]
Yeah I think this very likely doesn't work as expected. Does it work if you add:

```toml
[tool.ruff]
force-exclude = true
```

**Comment by VDuchauffour** [comment]
It solves indeed the problem! Thank you.

**Comment by charliermarsh** [comment]
Nice :)

I think I should probably add that to the built-in command-line arguments in the LSP, so I'll leave this open to track that.

**Comment by charliermarsh** [comment]
Closed by https://github.com/charliermarsh/ruff-lsp/pull/86.
